<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AdminWindow</class>
 <widget class="QMainWindow" name="AdminWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>AdminWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="2" column="5">
     <widget class="QPushButton" name="pushButton_reload">
      <property name="text">
       <string>Обновить</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0" colspan="7">
     <widget class="QListWidget" name="listWidget"/>
    </item>
    <item row="2" column="6">
     <widget class="QPushButton" name="pushButton_close">
      <property name="text">
       <string>Закрыть</string>
      </property>
     </widget>
    </item>
    <item row="0" column="6">
     <widget class="QComboBox" name="comboBox"/>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
